app/controllers/images_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ImagesController < ApplicationController
+
+  def index
+    @images = Image.all
+  end
+
+  def new
+    @image = Image.new
+  end
+
+  def create
+    Image.create(image_params)
+    redirect_to '/images'
+  end
+
+  def show
+    @image = Image.find(params[:id])
```

Comment:
You've got this line of code `@image = Image.find(params[:id])` quite a few times. Could you refactor this somehow?